src/server/mutate.rs

```diff
@@ -56,12 +102,50 @@ async fn mutate_internal(review: AdmissionReview) -> Result<impl warp::Reply, wa
         ));
     };
 
-    let patch = r#"[{
-        "op": "add",
-        "path": "/spec/runtimeClassName",
-        "value": "edera"
-    }]"#;
-    let patch = BASE64_STANDARD.encode(patch);
+    // Get name and namespace from the object's metadata.
+    let metadata = request.object.metadata;
+    let name = metadata.name.unwrap_or_else(|| {
+        metadata
+            .generate_name
+            .unwrap_or_else(|| "unknown".to_string())
+    });
+    let namespace = metadata.namespace;
+
+    // Prevent mutating resources in the kube-system namespace
```

Comment:
@denhamparry I'd prefer to lift this out to a configuration rather than being implicit inside the server itself. Eventually we may want to `kube-system` resources and it would be good for end users to see which namespaces are being excluded through the configuration rather than the logs.

I'm fine if we merge this and file an issue to add more configuration options